ietf/templates/doc/document_history.html

```diff
@@ -18,85 +18,7 @@
-                </div>
-            </div>
-            <div class="row mb-3">
-                <label for="url2" class="col-form-label col-sm-2 fw-bold">To revision</label>
-                <div class="col-sm-10">
-                    <select class="form-select select2-field" data-max-entries="1" data-minimum-input-length="0" id="url2" name="url2">
-                        {% for name, rev, time, url in diff_revisions %}
-                            <option value="{{ url }}"
-                                    {% if forloop.counter == 1 %} selected="selected"{% endif %}>
-                                {{ name }}
-                                {% if rev %}-{{ rev }}{% endif %}
-                                ({{ time|date:"Y-m-d" }})
-                            </option>
-                        {% endfor %}
-                        {% if doc.name|rfcbis %}
-                            <option value="{{ doc.name|rfcbis }}">
-                                {{ doc.name|rfcbis }}
-                            </option>
-                        {% endif %}
-                    </select>
-                </div>
-            </div>
-            <div class="row mb-3">
-                <label class="col-form-label col-sm-2 fw-bold">Diff format</label>
-                <div class="col-sm-10">
-                    <div class="btn-group" data-bs-toggle="buttons">
-                        <input type="radio"
-                               class="btn-check"
-                               checked
-                               name="difftype"
-                               value="--html"
-                               id="html">
-                        <label for="html" class="btn btn-outline-primary">Side-by-side</label>
-                        <input type="radio"
-                               class="btn-check"
-                               name="difftype"
-                               value="--abdiff"
-                               id="abdiff">
-                        <label for="abdiff" class="btn btn-outline-primary">Before-after</label>
-                        <input type="radio"
-                               class="btn-check"
-                               name="difftype"
-                               value="--chbars"
-                               id="chbars">
-                        <label for="chbars" class="btn btn-outline-primary">Change bars</label>
-                        <input type="radio"
-                               class="btn-check"
-                               name="difftype"
-                               value="--hwdiff"
-                               id="hwdiff">
-                        <label for="hwdiff" class="btn btn-outline-primary">Wdiff</label>
-                    </div>
-                </div>
-            </div>
-            <button type="submit" class="btn btn-primary mb-3">Submit</button>
-        </form>
+        {% include "doc/document_history_form.html" %}
```

Comment:
I'm not sure how Robert feels, but when including templates likely to be used in different views I think we ought to be passing the parameters as part of the `{% include %} instead of sharing the rendering context. I.e.,
```
{% include "<template path>" action_url=rfcdiff_base_url doc=doc blah=blah foo=bar only %}
```
That makes it much easier to sort out how to use the template in a new place.

Reply:
I like the idea